cypress/integration/pages/home.spec.js

```diff
@@ -64,4 +64,30 @@ context('Index page', () => {
     });
 
   });
+
+  describe("Topic Explorer", () => {
+    it("can show the topic explorer", () => {
+      cy.contains('How can we help?').should('be.visible')
+    })
+
+
+    it("can show food example prompts", () => {
+      cy.get('[data-search-term=food]').click();
+      cy.get('input').should('have.value', 'food')
+    })
+
+
+    it("can show mental health example prompts", () => {
+      cy.get("#example-search").contains('mental health').click();
```

Comment:
This assertion is done in different ways on lines 75 and 87. It's good to pick one consistent way, I think.

(I like this way of doing it a little better - by text rather than data-attributes - but I'm happy either way)

Reply:
I've changed it so it checks the text. I couldn't use the data attribute as a selector because `mental health` has a space so `'[data-search-term=mental health]'` doesnt work